README.md

```diff
@@ -56,56 +60,95 @@ To get started, create an override configuration for your ESLint, while applying
       "parser": "@graphql-eslint/eslint-plugin",
       "plugins": ["@graphql-eslint"],
       "rules": {
-        ...
+        "@graphql-eslint/require-description": [
```

Comment:
Added the most commonly applicable rule I could think of, such that this snippet would be enough to deliver value independently.

Unfortunately, the `require-description` rule config is quite verbose. I know @B2o5T has made some changes for the coming major release which reduce this verbosity. I'm curious if it could be reduced further to accept `["error"]` and just default to the complete set of definitions.

Reply:
I think we can add some simple rule like `no-anonymous-operations` instead.

And about `require-description` rule, I think this rule can't have a default set of definitions - because somebody wants to require description only for `fields`, somebody only for `types`, and somebody only for `ObjectTypeDefinition`.

Reply:
🤷‍♂️ Whatever defaults are easiest to expect for the user and also safely not violating any best practices. I imagine descriptions to all definitions is a safe default, and then folks can back it off from there or specify their own.